plugins/polio/js/src/domains/Campaigns/Dashboard/DashboardButtons.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent } from 'react';
+import { Box, Grid } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { useSafeIntl } from 'bluesquare-components';
+import DownloadIcon from '@mui/icons-material/GetApp';
+import { LinkButton } from '../../../components/Buttons/LinkButton';
+import { userHasPermission } from '../../../../../../../hat/assets/js/apps/Iaso/domains/users/utils';
+import { useCurrentUser } from '../../../../../../../hat/assets/js/apps/Iaso/utils/usersUtils';
+import MESSAGES from '../../../constants/messages';
+import { CreateCampaignModal } from '../MainDialog/CreateCampaignModal';
+import { Router } from '../../../../../../../hat/assets/js/apps/Iaso/types/general';
+
+const useStyles = makeStyles(theme => {
+    return {
+        marginBottom: { marginBottom: theme.spacing(0) },
```

Comment:
better use a Box with mb prop

Reply:
Question which is the most costly: adding a css rule, or adding a level of nesting in the dom with the Box?

I don't really mind either way. I thought I'd try this way to avoid adding a level of nesting just for a margin bottom